spec/models/note_spec.rb

```diff
@@ -24,14 +24,16 @@
     end
   end
 
-  describe '#to_s' do
+  describe '#to_csv' do
     before do
       subject.note = 'Test note'
       subject.created_at = 'Nov 3, 2011'
       allow(user).to receive_messages(name: 'user')
     end
 
-    its(:to_s) { should == 'Test note (user - Nov 03, 2011)' }
+    it 'should have the format "task_name (user_name - date)"' do
```

Comment:
Could you remove the should from this spec description?
http://www.betterspecs.org/#should

Reply:
What do think about: `gets formated task "task_name (user_name - date)"` is It better?

Reply:
@hjJunior you didn't check the case when the user aren't present, so you need to create another spec to verify.

Check it out and let me know if this works.

```rb
context 'When Note has a user' do
  let(:note) do
    build_stubbed(
      :note,
      user: user,
      note: 'Test note',
      created_at: 'Nov 3, 2011'
    )
  end

  before { allow(user).to receive_messages(name: 'user') }

  subject { note.to_csv }

  it { is_expected.to eq('Test note (user - Nov 03, 2011)') }
end

context 'When Note does not have a user' do
  let(:user_name) { I18n.t('author unknown') }

  let(:note) do
    build_stubbed(:note, note: 'Test note', created_at: 'Nov 3, 2011')
  end

  subject { note.to_csv }

  it { is_expected.to eq('Test note (#{user_name} - Nov 03, 2011)') }
end
```